tools/extractors/extract_battlescape_stuff.cpp

```diff
@@ -89,6 +150,13 @@ static void readBattleMapParts(GameState &state, BattleMapPartType::Type type,
 		object->imageOffset = {24, 48};
 
 		state.battle.map_part_types[id] = object;
+
+		if (false) // debug output to get images into files
```

Comment:
I guess this is for local-debugging, it probably won't be too useful for other users, so maybe remove before merging into git?